test/e2e/suite_test.go

```diff
@@ -87,6 +89,12 @@ const (
 	interval                            = time.Millisecond * 250
 )
 
+func TestAPIs(t *testing.T) {
```

Comment:
Hi @trdoyle81, the function definition was a little different before, any reason you changed it?

```
func TestAPIs(t *testing.T) {
	RegisterFailHandler(Fail)

	RunSpecsWithDefaultAndCustomReporters(t,
		"Controller Suite",
		[]Reporter{printer.NewlineReporter{}})
}
```

Reply:
The required import used with custom reporters, namely "sigs.k8s.io/controller-runtime/pkg/envtest/printer" is not available in this version of golang (v1.21).